main/zoe/guide/contracts/multipoolAutoswap.md

```diff
@@ -0,0 +1,177 @@
+# Multipool Autoswap
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/2a8b0fc2ece7344604bcc23b295367cd871f6995/packages/zoe/src/contracts/multipoolAutoswap/multipoolAutoswap.js) (Last updated: 2020-09-14)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+
+Multipool Autoswap is a rewrite of [UniSwap](https://uniswap.org/) that supports
+multiple liquidity pools and direct exchanges across pools.
+
+Each liquidity pool maintains a price for exchanges between the central token and a
+secondary token. Secondary tokens can be exchanged with each other, but only through
+the central token. For example, if X and Y are two token types and C is the central
+token, a swap giving X and wanting Y would first use the pool (X, C) then the pool
+(Y, C). There are no direct liquidity pools between two secondary tokens.
+
+There should only need to be one instance of this contract, so liquidity can be
+shared as much as possible. Each secondary currency has a separate pool of liquidity.
+
+When the contract is instantiated, the central token is specified in the
+terms. Separate invitations are available by calling methods on the publicFacet for
```

Comment:
Not clear on how invitations are separate? What is each invitation for?

Reply:
clarified